bluesky/tests/test_scans.py

```diff
@@ -166,13 +166,43 @@ def test_ascan(RE, hw):
     traj_checker(RE, scan, traj)
 
 
+def test_multi_motor_list_scan(RE, hw):
+    expected_data = [
+        {'motor2': 18, 'motor2_setpoint': 18, 'det': 1,
+         'motor1': 6, 'motor1_setpoint': 6},
+        {'motor2': 28, 'motor2_setpoint': 28, 'det': 1,
+         'motor1': 7, 'motor1_setpoint': 7},
+        {'motor2': 38, 'motor2_setpoint': 38, 'det': 1,
+         'motor1': 8, 'motor1_setpoint': 8}]
+
+    scan = bp.list_scan([hw.det], hw.motor1, [6, 7, 8],
+                        hw.motor2, [18, 28, 38])
```

Comment:
Why not to have numpy-arrays as inputs for this case as in the tests below?

Reply:
good catch, I test with lists in another test this was suppose to test with numpy arrays.